### ISSUE TITLE
Simplify script running so users don't have to write 'python'

### ISSUE THREAD

**Issue by carlhiggs** [healthysustainablecities/global-indicators#259]
Currently, when users launch into our software environment this looks something like the following:
```
>global-indicators.bat

Global Healthy and Sustainable City Indicators
4.2.1

https://healthysustainablecities.org ('1000 Cities Challenge')
https://global-healthy-liveable-cities.github.io/ (Software guide)

[+] Running 2/2
 - Container ghsci           Started                                                                                                                                                                        0.6s
 - Container ghscic_postgis  Running                                                                                                                                                                        0.0s

To run the provided example analysis for Las Palmas, Spain, enter:
python 1_create_project_configuration_files.py example_ES_Las_Palmas_2023
python 2_analyse_region.py example_ES_Las_Palmas_2023
python 3_generate_resources.py example_ES_Las_Palmas_2023

For more directions on each step, run the same command without including the study region codename.

To exit, type and enter: exit
```

In #257 we added an environment file `.env` that contains an instructions variable that is echoed when the container is launched.   I believe we could also create environment variables to simplify the instructions further --- like:

```
To run the provided example analysis for Las Palmas, Spain, enter:
$configure example_ES_Las_Palmas_2023
$analysis example_ES_Las_Palmas_2023
$generate example_ES_Las_Palmas_2023

For more directions on each step, run the same command without including the study region codename.

To exit, type and enter: exit
```

so the gist is to configure type "$configure [codename]" and likewise for the "$analysis" and "$generate" resources steps.   While the dollars sign is a bit wierd its the mechanism that allows this to work, as the underlying commands above can be stored in environment variables.

To do this, we expand on the .env file:
```
configure=python 1_create_project_configuration_files.py
analysis=python 2_analyse_region.py
generate=python 3_generate_resources.py
dollar="\$"
INSTRUCTIONS=\\nTo run the provided example analysis for Las Palmas, Spain, enter:\\n\${dollar}configure example_ES_Las_Palmas_2023\\n\${dollar}analysis example_ES_Las_Palmas_2023\\n\${dollar}generate example_ES_Las_Palmas_2023\\n\\nFor more directions on each step, run the same command without including the study region codename.\\n\\nTo see these instructions again, type: "\${dollar}help"\\nTo exit, type and enter: exit\\n\\n
UID=''
GID=''
```

and add the following to the ghsci container definition in docker-compose.yml:
```
    environment:    
        - configure
        - analysis
        - generate
```

In short, the benefit is the user doesn't need to remember obscure script names or python syntax - just three words: configure, analysis and generate.   

While 'analyse' or 'analyze' would be more consistent, saying 'analysis' sidesteps localisation issues with British/Australian and US English forms.   The prefix by a $ symbol is probably a bit obscure, but its easy enough to remember that this is required to issue the command.

The above works on a Windows computer; I am assuming it could on other platforms, but should test this.

**Comment by carlhiggs** [comment]
@gboeing , we just discussed this in a meeting and you made a good point about the balance between making simple and supporting technical users.  I agree - need to enable both.  We need to update our setup script to allow our scripts to be imported as libraries (eg in a juypyter notebook), without supplying a study region as a command line argument -- this is probably the way that more technical people (Python or R users, via reticulate) would like to approach this.   Just wanted to acknowledge and agree --- more work to do!

**Comment by gboeing** [comment]
@carlhiggs yes I think some combo of 1) traditional technical data scientist way of interacting with the workflow + 2) point and click (web?) user interface for people with minimal technical skills would be great.
